app/Http/Controllers/RequestWithdrawalApprovalController.php

```
@@ -0,0 +1,55 @@
+                ->additional([
+                    'message' => 'Request Withdrawal Approvals retrieved successfully.',
+                    'success' => true,
+                ])
+                ->response()
+                ->setStatusCode(200);
+        } catch (Throwable $e) {
+            Log::error('Error fetching Request Withdrawal Approvals', [
+                'user_id' => auth()->id(),
+                'error'   => $e->getMessage(),
+                'trace'   => $e->getTraceAsString(),
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'index'. (Clean Code Rules)

(StaticAccess)